product-service/handlers/getProductsList.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+const productsList = [
```

Comment:
I would recommend extracting this list and methods that manipulate this list into a separate place to fulfill the MVC pattern.

Reply:
Thanks for your comment. I'm intending to replace it with a real DB request in the next task.